launcher/ui/MainWindow.cpp

```diff
@@ -1342,16 +1342,10 @@ void MainWindow::on_actionExportInstanceFlamePack_triggered()
     if (m_selectedInstance) {
         auto instance = dynamic_cast<MinecraftInstance*>(m_selectedInstance.get());
         if (instance) {
-            QString errorMsg;
-            if (instance->getPackProfile()->getComponent("org.quiltmc.quilt-loader")) {
-                errorMsg = tr("Quilt is currently not supported by CurseForge modpacks.");
-            } else if (auto cmp = instance->getPackProfile()->getComponent("net.minecraft");
-                       cmp && cmp->getVersionFile() && cmp->getVersionFile()->type == "snapshot") {
-                errorMsg = tr("Snapshots are currently not supported by CurseForge modpacks.");
-            }
-            if (!errorMsg.isEmpty()) {
+            if (auto cmp = instance->getPackProfile()->getComponent("net.minecraft");
+                cmp && cmp->getVersionFile() && cmp->getVersionFile()->type == "snapshot") {
                 QMessageBox msgBox;
```

Comment:
```suggestion
                QMessageBox msgBox(this);
```
might as well do this now?